.github/workflows/auto-badge.yaml

```diff
@@ -0,0 +1,41 @@
+name: auto-badge
+
+on:
+  workflow_call:
+    secrets:
+      AUTOMATION_TOKEN:
+        required: true
+
+env:
+  REPO_NAME: ${{ github.event.repository.name }}
+  REPO_OWNER: ${{ github.repository_owner }}
+
+jobs:
+  auto-badge:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Git checkout
+      uses: actions/checkout@v3
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Git checkout Labs Tools
+      uses: actions/checkout@v3
+      with:
+        token: ${{ secrets.AUTOMATION_TOKEN }}
+        repository: boldlink/labs-tools-scripts
+        ref: 'main'
+        path: .terraform_ignore
```

Comment:
Woudn't it make more sense to change the name to `.git_ignore`?

Reply:
Also to be sure, whatever the folder we choose to use should be in the `.gitignore` file in the terraform-module-folder